### ISSUE TITLE
Request to document detailed procedure for  building the lambda layer

### ISSUE THREAD

**Issue by kaustubhd93** [aalvrz/python3-saml-aws-lambda-layer#1]
Thank you for creating the lambda layer. It works out of the box. Raising this issue in sincere request to document the procedure of building this lambda layer. 

## Please mention details like :
- pre-requisites
- OS
- How to find out which .so files are required in the lib folder
- how to build pip packages from source

**Comment by agsdot** [comment]
Yes @aalvrz , if you could please share how to create this lambda layer that would be greatly appreciated.
